.github/workflows/coverage_converter_publish_actions.yml

```diff
@@ -0,0 +1,134 @@
+name: Coverage Converter Publish Actions
+
+on:
+  push:
+    paths:
+      - 'metrics/coverage_converter/**'
+    branches:
+      - 'master'
```

Comment:
It seems like we should run tests and builds on each update of the pull request similar to [Metrics Web Application Actions](https://github.com/platform-platform/monorepo/blob/master/.github/workflows/web_app_actions.yml) and publish a new release only on push to master.